docs/spring-boot/app-starter.md

```diff
@@ -95,8 +95,8 @@ bdk-app:
         allowed-origins: "*" # list of allowed origins path pattern that be specific origins,
         allowed-credentials: true # Access-Control-Allow-Credentials response header for CORS request
         allowed-method: ["POST", "GET"] # list of HTTP methods to allow
-        allowed-headers: ["header1", "header2"] # list of headers that a request can list as allowed
-        exposed-headers: ["header1", "header2"] # list of response headers that a response can have and can be exposed
+        allowed-headers: "*" # list of headers that a request can list as allowed
```

Comment:
In the comment (only), give an example with `["header1", "header2"]`

Reply:
the documentation of exposedHeaders says: Note that "*" is not supported on this property.

Reply:
and it will throw an exception if used